charts/zitadel/templates/deployment.yaml

```diff
@@ -85,8 +86,11 @@ spec:
                 - name: Host
                   value: {{ .Values.zitadel.configmapConfig.ExternalDomain }}
               {{- end }}
-            initialDelaySeconds: 60
-            periodSeconds: 5
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds | default 60 }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds | default 5 }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold | default 3 }}
```

Comment:
IMO, it's more consistent to move all defaults to the values.yaml.

Reply:
okay, i can do that and update it. Just a second